### PR TITLE
[RHBA-512] RHBA image yamls needs corrected OSBS repository name and branch plus component label

### PR DESCRIPTION
**Pull request by errantepiphany**
Signed-off-by: David Ward <dward@redhat.com>

[RHBA-512] RHBA image yamls needs corrected OSBS repository name and branch plus component label
https://issues.jboss.org/browse/RHBA-512

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ x] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [x ] Pull Request contains link to the JIRA issue
- [x ] Pull Request contains description of the issue
- [x ] Pull Request does not include fixes for issues other than the main ticket
- [x ] Attached commits represent units of work and are properly formatted
- [x ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: controller/image.yaml [hunk 1]
```diff
@@ -40,5 +40,5 @@ run:
           - "standalone-full.xml"
 osbs:
       repository:
-          name: rhba-7-controller-docker
-          branch: jb-controller70
+          name: containers/rhba7-controller
+          branch: jb-rhba7.0-controller-rhel-7
```
Comment (goldmann) [inline controller/image.yaml]: You missed dash after rhba.

Comment (errantepiphany) [reply controller/image.yaml]: Okay, I think I've fixed it. Had to do it in kiesever, controller, and smartrouter.

Comment (errantepiphany) [reply controller/image.yaml]: @goldmann Better?

**Comment by goldmann** [comment]
Sorry, this is the standalone repo. Reviewing on phone sucks.

**Comment by goldmann** [comment]
All good!

**Pull request merged**